src/ui/ExportToExcel/ExportToExcel.ts

```diff
@@ -27,24 +27,25 @@ export class ExportToExcel extends Component {
   static options: IExportToExcelOptions = {
 
     /**
-     * Specifies the number of results to include in the exported Excel file.
+     * Specifies the number of results to include in the resulting Excel file.
      *
      * Generating and downloading the Excel file should take a reasonably short amount of time when using the default
-     * value. However, this amount of time will increase exponentially if you set the value higher.
+     * value. However, this amount of time will increase exponentially as you set the value higher.
      *
-     * It is therefore not recommended to set this value above the default index limit of 1000 search results.
+     * Consequently, you should avoid setting this value above the default index limit of 1000 search results.
      *
-     * Default value is `100`.
+     * Default value is `100`. Minimum value is `1`.
      */
     numberOfResults: ComponentOptions.buildNumberOption({ defaultValue: 100, min: 1 }),
     fieldsToInclude: ComponentOptions.buildFieldsOption()
```

Comment:
FYI : This is not documented because we are waiting on a fix from the Search API team to support it correctly.

So instead of documenting something that does not work flawlessly from end to end, we decided to remove it from the documentation for the time being.